src/App.tsx

```diff
@@ -1,24 +1,305 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+/* eslint-disable no-param-reassign */
+import React, {
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {
+  createTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+} from './api/todos';
+import { AuthContext } from './components/Auth/AuthContext';
+import { NewTodoField } from './components/NewTodoField/NewTodoField';
+import { TodoFilters } from './components/TodoFilters/TodoFilters';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [isLoadError, setIsLoadError] = useState(false);
+  const [isUploadError, setIsUploadError] = useState(false);
+  const [isRemoveError, setIsRemoveError] = useState(false);
```

Comment:
you have different state variables for errors. I would recommend creating `enum` for `ErrorType`. Then you can just have one state `[error, setError]` for different errors and show text according to the type.